### DISCUSSION TITLE
Random Lagging

### DISCUSSION THREAD

**Discussion by muju96** [wolveix/satisfactory-server#240] [Q&A] [answered]
I'm running the server on unraid with three friends playing at a time and keep getting lag jumps pretty consistently. Does this docker container need to have allocated resources to help avoid that? like set aside 24gb of ram at a minimum and reserve a set of cores for it? I'd blame my internet but I have gb and in my monitoring I'm not pushing my limits at all. very new to all this so any help or information is greatly appreciated.

**Comment by wolveix** [comment, marked as answer]
@muju96 this is pretty common with the Satisfactory server implementation in general, and the Docker container does not influence this. Try disabling the `SERVERSTREAMING` environment variable, though do note that this will consume significantly more RAM (allow 12GB+)

**Reply by muju96** [reply]
Ok cool, I felt as much since the dedicated server for satisfactory is relatively new and still getting worked on.
I'll try that disabling and go from there. In the end though really enjoying it so even with lag spikes still having an awesome time :)

**Reply by wolveix** [reply]
Glad to hear you're having fun with it! :D
